src/bioregistry/data/bioregistry.json

```diff
@@ -25396,7 +25396,7 @@
     "publications": [
       {
         "doi": "10.3233/sw-150177",
-        "title": "The Document Components Ontology (DoCO)",
+        "title": "The Document Components Ontology (DoCO)",
```

Comment:
A number of strings still have the spurious diffs like this. Could you revert these, or start again with the original bioregistry.json and just apply the specific changes needed to update URLs?